boards/blues/cygnet/doc/index.rst

```diff
@@ -0,0 +1,202 @@
+.. zephyr:board:: cygnet
+
+Overview
+********
+
+The Blues Cygnet board features an ARM Cortex-M4 based STM32L433CC MCU
+with a wide range of connectivity support and configurations. Here are
+some highlights of the Cygnet board:
+
+- STM32L4 microcontroller in LQFP48 package
+- Adafruit Feather connector
+- User LED
+- User push-button
+- USB Type-C connector
+
+More information about the board can be found at the `blues.dev website`_.
+
+Hardware
+********
+
+The STM32L433CC SoC provides the following hardware IPs:
+
+- Ultra-low-power with FlexPowerControl (down to 28 nA Standby mode and 84
+  |micro| A/MHz run mode)
+- Core: ARM |reg| 32-bit Cortex |reg| -M4 CPU with FPU, frequency up to 80 MHz,
+  100DMIPS/1.25DMIPS/MHz (Dhrystone 2.1)
+- Clock Sources:
+
+  - 32 kHz crystal oscillator for RTC (LSE)
+  - Internal 16 MHz factory-trimmed RC ( |plusminus| 1%)
+  - Internal low-power 32 kHz RC ( |plusminus| 5%)
+  - Internal multispeed 100 kHz to 48 MHz oscillator, auto-trimmed by
+    LSE (better than |plusminus| 0.25 % accuracy)
+  - 2 PLLs for system clock, USB, audio, ADC
+
+- RTC with HW calendar, alarms and calibration
+- 11x timers:
+
+  - 1x 16-bit advanced motor-control
+  - 1x 32-bit and 2x 16-bit general purpose
+  - 2x 16-bit basic
+  - 2x low-power 16-bit timers (available in Stop mode)
+  - 2x watchdogs
+  - SysTick timer
+
+- Up to 21 fast I/Os, most 5 V-tolerant
+- Memories
+  - Up to 256 KB single bank Flash, proprietary code readout protection
+  - 64 KB of SRAM including 16 KB with hardware parity check
+
+- Rich analog peripherals (independent supply)
+  - 1x 12-bit ADC 5 MSPS, up to 16-bit with hardware oversampling, 200
+    |micro| A/MSPS
+  - 2x 12-bit DAC output channels, low-power sample and hold
+  - 1x operational amplifiers with built-in PGA
+  - 2x ultra-low-power comparators
+
+- 17x communication interfaces
+  - USB 2.0 full-speed crystal less solution with LPM and BCD
+  - 1x SAI (serial audio interface)
+  - 3x I2C FM+(1 Mbit/s), SMBus/PMBus
+  - 4x USARTs (ISO 7816, LIN, IrDA, modem)
+  - 1x LPUART (Stop 2 wake-up)
+  - 3x SPIs (and 1x Quad SPI)
+  - CAN (2.0B Active)
+
+- 14-channel DMA controller
+- True random number generator
+- CRC calculation unit, 96-bit unique ID
+- Development support: serial wire debug (SWD), JTAG, Embedded Trace Macrocell*
+
+More information about STM32L433CC can be found here:
+
+- `STM32L433CC on www.st.com`_
+- `STM32L432 reference manual`_
+
+Supported Features
+==================
+
+.. zephyr:board-supported-hw::
+
+.. note:: CAN feature requires CAN transceiver
+
+Other hardware features are not yet supported on this Zephyr port.
+
+The default configuration can be found in the defconfig file:
+:zephyr_file:`boards/blues/cygnet/cygnet_defconfig`
+
```

Comment:
not really needed and we're trying to get rid of this wording across all boards' docs

```suggestion
```